linters/trufflehog/plugin.yaml

```diff
@@ -34,7 +34,23 @@ lint:
           cache_ttl: 1h
           parser:
             runtime: python
-            run: python3 ${plugin}/linters/trufflehog/trufflehog_to_sarif.py
+            run:
+              python3 ${plugin}/linters/trufflehog/trufflehog_to_sarif.py
+              --parse-mode=1-indexed-linenos
```

Comment:
optional: I would prefer just doing this exactly as we do it for `ruff` (passing in 1 or 0 for argv[1]), but this is alright

Reply:
I hear what you're saying about overkill, and it was definitely in the back of my mind, but I think this makes the plugin.yaml much more readable so I'm willing to make that tradeoff.

Reply:
Fair enough. Not a hill I'd die on, but I stand by my opinion. Fine to land this parser line as is